sessionctx/variable/varsutil_test.go

```diff
@@ -49,120 +45,120 @@ func (s *testVarsutilSuite) TestTiDBOptOn(c *C) {
 		{"1.1", false},
 		{"", false},
 	}
-	for _, t := range tbl {
-		on := TiDBOptOn(t.val)
-		c.Assert(on, Equals, t.on)
+	for _, tbl := range table {
+		on := TiDBOptOn(tbl.val)
+		require.Equal(t, tbl.on, on)
 	}
 }
 
-func (s *testVarsutilSuite) TestNewSessionVars(c *C) {
-	defer testleak.AfterTest(c)()
+func TestNewSessionVars(t *testing.T) {
+	defer testleak.AfterTestT(t)()
```

Comment:
after using `goleak.VerifyMain(m)`, you don't have to use this funcation.

ditto others.

Reply:
Fixed it with the following commit - 9f0a372c9660f842effd6baa9b939998914db125